Model/DutyCalculator.php

```diff
@@ -144,12 +159,12 @@ public function handleTaxApplicability($address, $apply)
         $this->_logger->debug("Value of isDutyOptional ".$isDutyOptional);
 
         if ($apply || !$isDutyOptional) {
-           $this->_logger->debug("Duty should be applied (due to user selection or the value of the 'apply' ".
-               " parameter that is passed to this function or because duty is not optional for the chosen country ".
-               "(shipment address)");
-           $quote->setReachDuty($this->checkoutSession->getReachDuty());
-           $quote->setBaseReachDuty($this->checkoutSession->getBaseReachDuty());
-           $quote->setDhlQuoteId($this->checkoutSession->getDhlQuoteId());
+            $this->_logger->debug("Duty should be applied (due to user selection or the value of the 'apply' ".
+                " parameter that is passed to this function or because duty is not optional for the chosen country ".
+                "(shipment address)");
+            $quote->setReachDuty($this->checkoutSession->getReachDuty());
+            $quote->setBaseReachDuty($this->checkoutSession->getBaseReachDuty());
+            $quote->setDhlQuoteId($this->checkoutSession->getDhlQuoteId());
```

Comment:
looks like just formatting changes

Reply:
I tried putting in explicit spaces, but on refresh, it still shows old formatting.